### PR TITLE
Enable getting Stomp authentication credentials from the environment

### PR DESCRIPTION
**Pull request by DiamondJoseph**
- Also removes the mounting of a manually formatted secret containing the credentials

### CODE REVIEW

**Comment by DiamondJoseph** [comment]
Considerations: these credentials are used to connect to whatever message bus is configured. 
Theoretically, if a  user had permissions to deploy to a namespace, but not to read Secrets in a namespace, and they knew that a secret was mounted as an envVar to this container, they could intercept the messaging. I do not think at Diamond we will ever have users who have deployment permissions to a namespace who don't also have permission to read its secrets. And even for users that *don't* have permission to deploy to the cluster, when reading mounted plans from dir, they can just put os.environ into the metadata of a plan and see it all anyway.

The RMQ passcode still isn't particularly secret of a secret. Currently we mount the Secret as a file, which again, can also be read from arbitrary mounted plan code!

File: helm/blueapi/values.yaml [hunk 1]
```diff
@@ -95,7 +103,7 @@ worker:
   stomp:
     auth:
       username: guest
-      passcode: guest
+      passcode: ${RABBITMQ_PASSWORD}
```
[the remark this answers is not in the document]
Comment (DiamondJoseph) [reply helm/blueapi/values.yaml:106]: If the envVar isn't set, it'll default to "".
But Kubernetes will be unhappy about the lack of the secret to mount as the envVar before it got to that.

[the remark this answers is not in the document]
Comment (DiamondJoseph) [reply helm/blueapi/values.yaml:106]: I'm happy to revert the default creds and comment out the extraEnvVar for the RMQ password

[the remark this answers is not in the document]
Comment (DiamondJoseph) [reply helm/blueapi/values.yaml]: Restored to working by default, leaving the extraEnvVar commented out as a hint for later use

**Review by callumforrester — APPROVED** [review]

**Pull request merged**